docs/reference/environment-variables.rst

```diff
@@ -61,6 +61,21 @@ Environment variables used by meson-python
 .. _cross build definition file: https://mesonbuild.com/Cross-compilation.html
 .. _cibuildwheel: https://cibuildwheel.readthedocs.io/en/stable/
 
+.. envvar:: IPHONEOS_DEPLOYMENT_TARGET
+
+   This environment variable is used to specify the target iOS platform version
+   to the Xcode development tools.  If this environment variable is set,
+   ``meson-python`` will use the specified iOS version for the Python wheel
+   platform tag, instead than the iOS platform default of 13.0.
+
+   This variable must be set to a major/minor version (e.g., ``13.0`` or
+   ``15.4``).
+
+   Another way of specifying the target iOS platform is to use the
+   ``-mios-version-min`` compile and link flags.  However, it is not possible
+   for ``meson-python`` to detect this, and it will not set the Python wheel
+   platform tag accordingly.
```

Comment:
I would either remove this paragraph (as it does not add any information useful to meson-python users) or rephrase it: starting with stating that it exist another way to do something, and concluding noting that it does not work is not good style for documentation.

Reply:
This paragraph is a modified version of the one that already exists for `MACOSX_DEPLOYMENT_TARGET`; however, I agree it's worded awkwardly given the context. I'll clean both up.